v2.0/install-cockroachdb.html

```diff
@@ -83,13 +83,14 @@
 <p>See <a href="../releases/{{page.release_info.version}}.html" data-eventcategory="mac-releasenotes-download">Release Notes</a> for what's new in the latest release, {{ page.release_info.version }}. </p>
 
 <div id="mac-installs" class="clearfix install-wrap">
-<a href="#use-homebrew" class="install-button mac-button current" data-eventcategory="buttonClick-doc-install" data-eventaction="mac-homebrew">Use <div class="c2a">Homebrew</div></a>
-<a href="#download-the-binary" class="install-button mac-button" data-eventcategory="buttonClick-doc-install" data-eventaction="mac-binary">Download the <div class="c2a">Binary</div></a>
+<!-- <a href="#use-homebrew" class="install-button mac-button current" data-eventcategory="buttonClick-doc-install" data-eventaction="mac-homebrew">Use <div class="c2a">Homebrew</div></a> -->
```

Comment:
I believe you can also use a conditional to avoid needing to update this whenever a version is promoted to stable:

```
{% if page.version.stable? %}
```